### ISSUE TITLE
root:USER_AGENT environment variable warning with v1.0.0a46 during startup

### ISSUE THREAD

**Issue by msmygit** [langflow-ai/langflow#2085]
**Describe the bug**
When starting up a46 version, we get this warning, but there is no docs to understand how to properly set this variable,
```
% python -m langflow run --env-file ~/langflow/.env
Starting Langflow v1.0.0a46...
WARNING:root:USER_AGENT environment variable not set, consider setting it to identify your requests.
╭───────────────────────────────────────────────────╮
│ Welcome to ⛓ Langflow                             │
│                                                   │
│                                                   │
│ Collaborate, and contribute at our GitHub Repo 🚀 │
│                                                   │
│ Access http://127.0.0.1:7860                      │
╰───────────────────────────────────────────────────╯


```

**Browser and Version**
n/a

**To Reproduce**
1. Install the latest alpha
2. start up langflow

**Screenshots**
n/a

**Additional context**
n/a

**Comment by YzzzMao** [comment]
> To set the `USER_AGENT` environment variable correctly and avoid the warning when starting up Langflow v1.0.0a46, you need to add the `USER_AGENT` variable to your `.env` file. Here is an example of how you can set it:
> 
> ```dotenv
> USER_AGENT=YourUserAgentString
> ```
> 
> Replace `YourUserAgentString` with the appropriate user agent string you want to use. Add this line to your existing `.env` file [[1]](https://github.com/logspace-ai/langflow/blob/dev/deploy/.env.example).
> 
> _To continue talking to [Dosu](https://dosu.dev), mention @dosu._

how can i get my useragent?

**Comment by nicoloboschi** [comment]
Resolved in LangFlow 1.0.x
